blockly-rtc/src/index.js

```diff
@@ -33,15 +35,20 @@ document.addEventListener('DOMContentLoaded', () => {
       });
   const workspaceClient = new WorkspaceClient(
       workspace.id, getEvents, writeEvents);
-
   workspaceClient.listener.on('runEvents', (eventQueue) => {
     runEvents_(eventQueue);
   });
-
   workspaceClient.initiateWorkspace();
 
+  const markerManager = new MarkerManager(workspace.id, sendMarkerUpdate,
```

Comment:
Why do you import these functions here and then pass them through to the marker manager, instead of just importing them there?

Reply:
I want to be able to make MarkerManager an interface that supports markers for realtime collaboration regardless of how the handlers are implemented. Index.js will be the glue where developers can create an instance of MarkerManager by passing in their own handlers.

This is also how WorkspaceClient is implemented.